test/distributed/org/apache/cassandra/distributed/impl/AbstractCluster.java

```diff
@@ -100,6 +104,10 @@
 
     private final File root;
     private final ClassLoader sharedClassLoader;
+    private final int subnet;
+    private final TokenSupplier tokenSupplier;
+    private final Map<Integer, NetworkTopology.DcAndRack> nodeIdTopology;
+    private final Consumer configUpdater;
```

Comment:
Raw consumer type, should be `Consumer<IInstanceConfig>` for safety.

Reply:
good catch, for some reason IntelliJ hasn't been good with generics and this class when auto defining the fields =(